charmcraft.yaml

```
@@ -41,6 +42,8 @@ provides:
     interface: grafana_dashboard
   auth-devices-keys:
     interface: auth_devices_keys
+  probes:
+    interface: blackbox_exporter_probes
```

Nit: it could be nice to have a `description` field outlining what this is about. For example:
```suggestion
    interface: blackbox_exporter_probes
    description: |
      Send over to blackbox exporter the endpoints we want it to probe.
```